COMDOX-1734: Updated SCP topics for clarity (#941)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/storefront-compatibility/index.mdx

### B2C Storefront Compatibility Package

<TableWrapper nowrap={[2]}>

| If you use | Installation | What to do next |
|---|---|---|
| Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer | Adobe, automatically | [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) <br /> [Adobe Commerce 2.4.8](/reference/storefront-compatibility/v248/) |
| Commerce PaaS or on-premises with Adobe Commerce Optimizer | You, with Composer | [Manual installation](/setup/configuration/storefront-compatibility/install/) |
| Commerce PaaS or on-premises without Adobe Commerce Optimizer | Not available without Adobe Commerce Optimizer | [Backend options](/get-started/backends/) <br /> [Licensing requirements](/licensing/) |

</TableWrapper>

### Storefront Compatibility B2B Package

> B2B drop-ins need this package in addition to the B2C Storefront Compatibility Package. The B2B package is available only on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer managed backends, not on Commerce PaaS or on-premises, even when you use Adobe Commerce Optimizer with PaaS for B2C drop-ins. Adobe manages it on Commerce when your instance includes Adobe Commerce B2B. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.

Reviewer: ```suggestion
B2B drop-ins need this package in addition to the B2C Storefront Compatibility Package. The B2B package is available only on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer with Commerce PaaS or on-premises. Adobe manages it on Commerce when your instance includes Adobe Commerce B2B. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.
```

---
title: Storefront Compatibility Package
description: Learn what the Storefront Compatibility Package does and find the installation path for your Commerce backend.
sidebar:
  label: Overview
  order: 1
---
import Link from '@components/Link.astro';
import TableWrapper from '@components/TableWrapper.astro';

> The Storefront Compatibility Package extends Adobe Commerce with GraphQL and REST operations that storefront drop-ins need for cart, checkout, account, and order flows. Use the tables below to find the installation path for each package.

Reviewer 2: The first sentence implies that cart, checkout, etc. are the only systems that benefit from the GraphQL enhancements. Expand the scope by saying "such as" or similar verbiage.

## Find your path

> The B2C Storefront Compatibility Package supports Adobe Commerce 2.4.8 and 2.4.9. For new storefront projects, Adobe Commerce 2.4.9 is recommended. Magento Open Source is not supported. If you run Adobe Commerce 2.4.7, see [Adobe Commerce 2.4.7](/reference/storefront-compatibility/v247/) instead. For supported versions and end-of-life dates, see <Link href="https://experienceleague.adobe.com/en/docs/commerce-operations/release/versions" text="Adobe Commerce Software Lifecycle Policy" />.

Reviewer 2: Be careful about versioning. ACCS is versionless.

### B2C Storefront Compatibility Package

<TableWrapper nowrap={[2]}>

| If you use | Installation | What to do next |
|---|---|---|

> | Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer | Adobe, automatically | [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) <br /> [Adobe Commerce 2.4.8](/reference/storefront-compatibility/v248/) |

Reviewer 2: Remove Optimizer from this line. It is a different beast entirely, and it doesn't install automatically. It's covered in the next line.

The What to do next column doesn't necessarily make a lot of sense for ACCS. Why are you sending them to 2.4.9 and 2.4.8? To read release notes?

With ACO, you probably need to mention the Connector. Optimizer-only (without Commerce) customers don't need our storefront.

| Commerce on Cloud or on-premises with Adobe Commerce Optimizer | You, with Composer | [Manual installation](/setup/configuration/storefront-compatibility/install/) |
| Commerce on Cloud or on-premises without Adobe Commerce Optimizer | Not available without Adobe Commerce Optimizer | [Backend options](/get-started/backends/) <br /> [Licensing requirements](/licensing/) |

</TableWrapper>

### Storefront Compatibility B2B Package

> B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with additional GraphQL and REST APIs as a separate package with its own release schedule. Adobe Commerce B2B must already be on your Commerce instance before you install SCP-B2B, and SCP-B2B supports only Adobe Commerce 2.4.8 and 2.4.9. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.

Reviewer 2: ```suggestion
B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with additional GraphQL and REST APIs as a separate package with its own release schedule. Adobe Commerce B2B must already be configured on your Commerce instance before you install SCP-B2B, and SCP-B2B supports only Adobe Commerce 2.4.8 and 2.4.9. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.
```

> B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with storefront-centric GraphQL. That makes it a separate package with its own release lifecycle. Adobe Commerce B2B must be on your Commerce instance before you install the package. SCP-B2B supports Adobe Commerce 2.4.8 and 2.4.9 only. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.

Reviewer 2: All GraphQL is storefront centric.

```suggestion
B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with additional GraphQL APIs. That makes it a separate package with its own release lifecycle. Adobe Commerce B2B must be on your Commerce instance before you install the package. SCP-B2B supports Adobe Commerce 2.4.8 and 2.4.9 only. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.
```

> B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with additional GraphQL APIs. That makes it a separate package with its own release lifecycle. Adobe Commerce B2B must be on your Commerce instance before you install the package. SCP-B2B supports Adobe Commerce 2.4.8 and 2.4.9 only. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.

Reviewer 2: ```suggestion
B2B drop-ins need the Storefront Compatibility B2B Package (SCP-B2B) in addition to the B2C Storefront Compatibility Package. SCP-B2B extends core Adobe Commerce B2B with additional GraphQL APIs. That makes it a separate package with its own release lifecycle. Adobe Commerce B2B must be enabled on your Commerce instance before you install the package. SCP-B2B supports Adobe Commerce 2.4.8 and 2.4.9 only. See [What you need](/setup/configuration/storefront-compatibility/b2b/#what-you-need) on the B2B package page for prerequisites.

```

<TableWrapper nowrap={[2]}>

| If you use | Installation | What to do next |
|---|---|---|

> | Adobe Commerce as a Cloud Service or Adobe Commerce Optimizer | Adobe, automatically | [B2B Compatibility Package](/setup/configuration/storefront-compatibility/b2b/) |

Reviewer 2: same as above
